onnx/defs/quantization/defs.cc

```diff
@@ -164,9 +171,9 @@ ONNX_OPERATOR_SET_SCHEMA(
           {// nodes: {outputs, op, inputs, attributes}
            FunctionBodyHelper::Const<float>("Q_Min", 0.f),
            FunctionBodyHelper::Const<float>("Q_Max", 255.f),
-           {{"X_Min"}, "ReduceMin", {"x"}, {MakeAttribute("keepdims", int64_t(0))}},
+           {{"X_Min"}, "ReduceMin", {"x"}, {MakeAttribute("keepdims", int64_t(0)), MakeRefAttribute("axes", "axis", AttributeProto::INTS), MakeAttribute("complement_axis", int64_t(1))}},
```

Comment:
The attribute which DynamicQuantizeLinear is using was AttributeProto::INT, but the attribute which Reduce is using is AttributeProto::INTS... In that case, it seems that we cannot use MakeRefAttribute directly here and the conversion from INT to INTS is needed. Does current FunctionBodyHelper support it?

Reply:
If not, I am thinking that maybe we can introduce other attribute (single integer) to represent the specified single quant dimension in reduce function instead of "complement_axis" (boolean).

Reply:
If we use a ContextDependentFunctionBody, we can do it. See https://github.com/onnx/onnx/blob/a7a0fec7f25cae567429af62b7eaaee1c3f0e247/onnx/defs/math/defs.cc#L2830 for an example. The context allows us to get the attribute-value, and use it to construct new attribute-values.

Reply:
@gramalingam I have just sent an email and haven't seen you replied here... Thank you so much and I will give this a try.

Reply:
Add attribute to QuantizeLinear node as well :)

Reply:
What do you mean by adding attribute to QuantizeLinear node?